docs/changelog/latest.md

```diff
@@ -1,9 +1,33 @@
 ---
 id: latest
-title: Changelog (v5)
+title: Changelog (v6)

```

Comment:
still v5.

Reply:
oh I thought the version would go to `next.6`.

Reply:
next6 !== v6. The version here tracks the major version. The expected official production version for neo3 is v5.0.0 .